lib/codebreaker.rb

```diff
@@ -4,7 +4,7 @@
 require 'yaml'
 require 'pry'
```

Comment:
This will lead to crash if we are not going to add `pry` gem in our `Gemfile` explicitly. My suggestion - remove this line.